priceservice/src/app.controller.ts

```diff
@@ -38,6 +43,8 @@ export class AppController {
    * The request sent by the account service will go to this endpoint and
    * will be transferred to the database service. A timeout interceptor is also
    * bind to this router handler that will throw a timeout exception after 5 seconds.
```

Comment:
außerdem: unklarer satzbaut. bezieht sich der nebensatz ("that will throw...") auf den interceptor oder auf den router handler? 
```suggestion
   * will be transferred to the database service. A timeout interceptor is also
   * bound to this router handler that will throw a timeout exception after 5 seconds.
```